core/src/main/resources/lib/hudson/executors.jelly

```diff
@@ -143,7 +143,7 @@ THE SOFTWARE.
                                       and !builtInHasExecutors)}"/>
   <!-- Remove the built-in from the list when it is included but has no executors to show -->
   <j:choose>
-    <j:when test="${!builtInHasExecutors and computers.get(0).node == app and computers.size() gt 1}">
+    <j:when test="${!builtInHasExecutors and computers.size() gt 1 and computers.get(0).node == app}">
```

Comment:
don't you want gt 0?
```suggestion
    <j:when test="${!builtInHasExecutors and computers.size() gt 0 and computers.get(0).node == app}">
```

Reply:
It should be gt 1, otherwise it fails when visiting the built-in executor page when that has 0 executors

Reply:
Is gt not greater than 0?

Reply:
It must be `gt 1` otherwise you get an `IndexOutOfBoundsException` in line 158 when visiting the built-in agent with 0 executors. In that case have a singleExecutor and we don't want the `computers` to be empty which would happen with `gt 0`